pkgs/desktops/gnustep/make/gsmakeDerivation.nix

```diff
@@ -1,7 +1,7 @@
 { lib, stdenv, make, makeWrapper, which }:
-{ buildInputs ? [], ...} @ args:
+{ nativeBuildInputs ? [], ...} @ args:
 stdenv.mkDerivation (args // {
-  buildInputs = [ makeWrapper make which ] ++ buildInputs;
+  nativeBuildInputs = [ makeWrapper make which ] ++ nativeBuildInputs;
```

Comment:
No need to support buildInputs here?

Reply:
the input arguments have `... } @ args` and mkDerivation has `(args // {` so no need to add it separately